src/uct/tcp/tcp_iface.c

```diff
@@ -872,7 +872,7 @@ int uct_tcp_keepalive_is_enabled(uct_tcp_iface_t *iface)
 {
 #ifdef UCT_TCP_EP_KEEPALIVE
     return (iface->config.keepalive.idle != UCS_TIME_INFINITY) &&
-           (iface->config.keepalive.cnt != 0) &&
+           (iface->config.keepalive.cnt != UCS_ULUNITS_INF) &&
```

Comment:
why changed?

Reply:
since, 0 is valid value and means no need to retry, fail immediately.
we should use `inf` as for other configuration parameters.

Reply:
need to document this in config param desc?

Reply:
done